requirements.txt

```diff
@@ -4,13 +4,18 @@ async-timeout==3.0.0
 attrs==18.1.0
 autobahn==18.6.1
 Automat==0.7.0
+certifi==2018.4.16
 channels==2.1.2
 channels-redis==2.2.1
+chardet==3.0.4
 constantly==15.1.0
 coverage==4.5.1
+coveralls==1.3.0
 daphne==2.2.0
 Django==2.0.6
+django-environ==0.4.5
```

Comment:
There seem to be some additions which are not part of this PR

Reply:
This came with the `pip3 freeze` command which I had forgotten to do before the last pull request.